### PR TITLE
made index.Rmd use data/ch path instead of daily/

### PR DESCRIPTION
**Pull request by karoliskoncevicius**
Only changed the index.Rmd (didn't upload the updated index.html, as I assume this is rebuild automatically).

We need to make this change in order to accommodate data from other countries, as currently all the indices are hosted under a single directory (daily/) with no prefix/suffix to denote the location.

### CODE REVIEW

File: docs/index.Rmd [hunk 1]
```diff
@@ -904,15 +904,15 @@ The category *Travel Abroad* includes search terms used to book flights and holi
...
-[Download data](https://raw.githubusercontent.com/trendecon/data/master/daily/travel_sa.csv){target="_blank"}
+[Download data](https://raw.githubusercontent.com/trendecon/data/master/data/ch/travel_sa.csv){target="_blank"}
 
 ## Row
 
 ### {style="border: 0px; background: none"}
 
 We see a similar and continuing slump in cultural events. In contrast, demand for food deliveries or DIY stores is at a (positive) record level.
 
-In summary, **trendEcon** offers a platform for daily updated information on the state of the Swiss economy. In the coming weeks, we will be adding further search terms and (sub-) indicators in order to constantly increase the amount of information available. In addition, we are constantly testing the usefulness of our data for forecasting economically relevant variables. All data are freely available for [Download](https://github.com/trendecon/data/tree/master/daily) and the code will be publicly available soon.
```
Comment (christophsax) [inline docs/index.Rmd:915]: > and the code will be publicly available soon.

Something to update at some point. @angelicambg or @KKPMW

Comment (karoliskoncevicius) [reply docs/index.Rmd:915]: ~Hello @christophsax , what update do you have in mind? The URL should be changed from /daily to /data/ in the pull req. (for some reason that part is not highlighted in the diff). Or do you mean the part about the availability of the code?~

Comment (karoliskoncevicius) [reply docs/index.Rmd:915]: Ah sorry, missed your quoted text.

**Comment by karoliskoncevicius** [comment]
Hello Christoph,

Would you have a few minutes for a chat today?

I am stuck between various pull request approvals, but have some time today and thinking if there is something I can do.

**Comment by christophsax** [comment]
In 3 min?

**Comment by karoliskoncevicius** [comment]
Just now saw the email, sorry! Still online?
